README.md

```diff
@@ -1,5 +1,7 @@
 # Math Magicians
 
+![screenshot](images/screenshot.png)
+
 ## Usage
 
 start by cloning this repository by running:
```

Comment:
- [OPTIONAL ]  Your README looks great 👍🏻  but I think you can improve it by making it more professional. A professional README file should have a project title, project description, list of technology used in the project, instructions to set the project up locally ... refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.